### PR TITLE
Add links at bottom of page and pics on /about/staff

### PR DESCRIPTION
**Pull request by seanlobo**
Added some links so that it's easier for people to learn more about the software listed if they want to.

### CODE REVIEW

File: ocfweb/templates/base.html [hunk 1]
```diff
@@ -186,7 +186,7 @@
                 {% block footer-top %}
                     <div class="ocf-footer-top">
                         <div class="container">
-                            <p class="text-center">OCF &hearts; Linux, Python, Puppet, Ruby, and Free/Open-Source Software. <span class="nowrap">Sound like you? <a href="{% url 'about-staff' %}">Join the staff team!</a></span></p>
+                            <p class="text-center">OCF &hearts; <a href="https://www.linuxfoundation.org/">Linux</a>, <a href="https://www.python.org/">Python</a>, <a href="https://puppet.com/">Puppet</a>, <a href="https://www.ruby-lang.org/en/">Ruby</a>, and Free/Open-Source Software. <span class="nowrap">Sound like you? <a href="{% url 'about-staff' %}">Join the staff team!</a></span></p>
```
Comment (jvperrin) [inline ocfweb/templates/base.html]: Might want to split this up into multiple lines?

**Comment by jvperrin** [comment]
@seanlobo I think this is just waiting on the whitespace fixes and then it can be merged! :smile: (No rush on that of course, it's finals currently)

**Comment by jvperrin** [comment]
RIP death was down, retriggering the build

**Comment by jvperrin** [comment]
Whoops, should have let @seanlobo  merge it, sorry about that... :slightly_frowning_face:

**Comment by seanlobo** [comment]
@jvperrin It's no big deal, thanks for helping me fix it!

**Pull request merged**